quickwit/quickwit-cli/tests/helpers.rs

```diff
@@ -101,16 +101,6 @@ pub fn make_command(arguments: &str) -> Command {
     cmd
 }
 
-pub fn make_command_with_list_of_args(arguments: &[&str]) -> Command {
```

Comment:
We still want to disable telemetry while running the tests.

Reply:
I think telemetry is only enabled when you run the CLI command (since the telemetry loop is only started on main()). When we don't execute the CLI and just call the underlying fn, I don't think it sends any telemetry data.